src/utils/github-utils.ts

```diff
@@ -239,14 +240,47 @@ const updateReadmeAndAutoMerge = async (repositoryOBJ: Repository, footer: strin
 export const updateReadmeAndAutoMergeRepositories = async (repositoriesOBJ: Repository[], footer: string) => {
   console.log("This script will add custom footers to ALL the specified repositories. It can also overwrite existing footers if desired.");
   console.log(chalk.red("If you do not want to automatically update ALL of the repositories selected, YOU MUST select 'n' in the following prompt."));
-  const processAllAnswer = promptSync(chalk.red("Do you want to add custom footer to ALL found repositories? (otherwise will be asked individually) (y/N): "));
+  
+  const processAllAnswer = promptSync(chalk.red("Do you want to add custom footer to ALL found repositories? (otherwise will be asked individually) (y/n/l for listing only repositories without a footer/q to quit): "));
+  if (processAllAnswer?.toLowerCase() === "q") {
+    console.log('Quitting the script...');
+    process.exit();
+  }
   const processAll = processAllAnswer?.toLowerCase() === "y";
+  const listOnlyWithoutFooter = processAllAnswer?.toLowerCase() === "l";
 
   if (processAll) {
     console.log("All repositories will be processed.");
   }
 
-  const overwriteAnswer = promptSync(chalk.red("If found do you want to automatically overwrite ALL existing metatavu-custom-footers (otherwise will be asked individually) (y/N): "));
+  if (listOnlyWithoutFooter) {
```

Comment:
updateReadmeAndAutoMergeRepositories function is getting bit long with the new code you have added. If it makes sense to you it might be worth refactoring it e.g. putting some related functionality into a separate function. This is not essential and perhaps isn't necessary- just a thought.